tests/test_05.R

```diff
@@ -0,0 +1,19 @@
+# copyright (c) - HU-Berlin / UTHSC / JICS by Danny Arends
+
+# Compare MPCC versus cor() function, mA = 15 x 5, mB = 15 x 10, no missing data
+library(MPCC)
+MPCCinfo()
+
+set.seed(1)
+mAB <- genAB(p = 5, n = 15, m = 10, missing = 1)
+
+ref <- cor(mAB[["A"]], mAB[["B"]], use="pair")
+mpcc <- PCC(mAB[["A"]], mAB[["B"]], debugOn=TRUE)
+
+mAB[["A"]]
```

Comment:
Could you disable the printing of the matrices ?
There is no real need to view them/ print them to the output every time

Danny

Reply:
Sure